src/org/labkey/remoteapi/query/ExecuteSqlCommand.java

```diff
@@ -318,7 +319,7 @@ public JSONObject getJsonObject()
     {
         JSONObject json = new JSONObject();
         json.put("schemaName", getSchemaName());
-        json.put("sql", getSql());
+        json.put("sql", EncodeUtils.wafEncode(getSql()));
```

Comment:
Not sure it's worth it, but we could make it possible to opt-out if we want to offer compatibility with older server versions.

Reply:
Yeah, I debated. I'm planning to release this as 6.0.0 to try to make clear that there's a compatibility break. But it wouldn't hurt to have an opt out.

Reply:
Added this for `ExecuteSqlCommand` but not `SqlExecuteCommand`, since I'm sure no one uses the latter